SSH.md

```diff
@@ -0,0 +1,17 @@
+(Note to team: should we cover PuTTy?)
+
+To work on the High Performance Computer (HPC), you need to connect remotely to it from your own computer.
+For this, you need to use SSH
+
+Skill set #1: Minimum requirements
+ * Understanding the syntax of an ssh command
+ * -Y option
+
+Skill set #2: Automate things
+ * ssh keys: generate and use
+ * ssh config file: what it is and some useful tidbits to put in yours
+ * ssh agent
+
+Skill set #3: Access your remote files easily
+ * sshfs
+ * (Note to team: What else for Windows machines?)
```

Comment:
Copying data - scp, rsync, bbcp, sftp

Reply:
Well, I think for now we should focus on listing out everything we think people should know. Then work out whether they'll get it from a training by us or other resources. We could simply link to resources instead of getting them into a training. I don't think we need to provide specific training on everything they need to know. A resource list might be just as effective (as long as this doc "tells" them: you don't know this, then go read that).